src/__test__/pages/experiments/[experimentId]/data-exploration/components/gene-list-tool/ListSelected.test.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```

Comment:
move this test file to `generic-gene-table` folder so that tests reflect correctly the structure of the files